quarkus-extension/engine/deployment/src/main/java/org/camunda/bpm/quarkus/engine/extension/deployment/impl/CamundaEngineProcessor.java

```diff
@@ -60,11 +66,19 @@ protected void unremovableBeans(BuildProducer<UnremovableBeanBuildItem> unremova
     unremovableBeansProducer.produce(UnremovableBeanBuildItem.beanTypes(CdiStandaloneProcessEngineConfiguration.class));
   }
 
+  @BuildStep
+  ContextConfiguratorBuildItem registerBusinessProcessScoped(ContextRegistrationPhaseBuildItem phase) {
+    return new ContextConfiguratorBuildItem(phase.getContext()
+        .configure(BusinessProcessScoped.class)
+        .normal()
+        .contextClass(InjectableBusinessProcessContext.class));
+  }
+
   @BuildStep
   protected void additionalBeans(BuildProducer<AdditionalBeanBuildItem> additionalBeansProducer) {
     additionalBeansProducer.produce(
         AdditionalBeanBuildItem.builder()
-            .setDefaultScope(DotNames.APPLICATION_SCOPED)
+            .setDefaultScope(DotName.createSimple(Dependent.class.getName()))
```

Comment:
This was not configured correctly. The default "implicit" scope of WELD is `Dependent`. Let's use this scope for these classes in Quarkus as well.